src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -811,7 +811,7 @@ module Mutations = struct
         | Ok
             ( `Broadcasted
             , Network_pool.Snark_pool.Resource_pool.Diff.Add_solved_work _
-            , _ ) ->
+            , () ) ->
```

Comment:
```suggestion
            , _ ) ->
```
It looks like this change triggers an error when building:

```
File "src/lib/mina_graphql/mina_graphql.ml", line 814, characters 14-16:
814 |             , () ) ->
                    ^^
Error: This pattern matches values of type unit
       but a pattern was expected which matches values of type
         Network_pool.Snark_pool.Resource_pool.Diff.rejected
```

Reply:
done!